docs/AA--Javascript-in-EE/asset-caching-by-filename.md

```diff
@@ -6,30 +6,76 @@ We also briefly considered using `filemtime` as the "version" generator for our
 
 So we've decided to [switch to filenames containing a hash](https://github.com/eventespresso/event-espresso-core/pull/287) generated from the content of the files.  This means if the content of a built file does not change, then the filename remains the same, if the content of a built bundle file changes, then the filename will as well.
 
-Switching to this new process requires some tweaks to how our assets are registered with `wp_register_script` however.
+Switching to this new process requires some tweaks to where assets are located, and how they are registered with `wp_register_script`.
+
+## Where Javascript and CSS Assets Should be Located
+
+In Event Espresso core, new Javascript and CSS files should be located somewhere under `assets/src` (previously they were placed in `core/templates/global_assets`, or beside the PHP files they corresponded to.)
+Put the files in whatever folder/subfolder is logical, like `assets/src/empire/stardestroyer.js'.
+`
```

Comment:
should be:

```
Put the files in whatever folder/subfolder is logical, like `assets/src/empire/stardestroyer.js`
```

(note backtick on next line)

Reply:
ahhh thanks